hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/contract/ContractTestUtils.java

```diff
@@ -1115,8 +1122,9 @@ public static void validateVectoredReadResult(List<FileRange> fileRanges, byte[]
     for (FileRange res : fileRanges) {
       CompletableFuture<ByteBuffer> data = res.getData();
       try {
-        ByteBuffer buffer = FutureIOSupport.awaitFuture(data);
-        assertDatasetEquals((int) res.getOffset(), "vecRead", buffer, res.getLength(), DATASET);
+        ByteBuffer buffer = FutureIO.awaitFuture(data);
```

Comment:
add a timeout here for better failure messages on a timeout.

Reply:
thanks